paper/paper.md

```diff
@@ -0,0 +1,190 @@
+Regularization works by adding penalty terms that penalize the model parameters in
+a variety of different ways. This can be used to incorporate prior knowledge 
+about the parameters in a structured form. In Pyglmnet, we offer
+users the ability to combine different types of regularization with different noise
+distributions in the GLMs.
+
+Despite the attractiveness of regularized GLMs, the available tools in
+the Python data science eco-system are highly fragmented. Specifically:
+
+-  [statsmodels] provides a wide range of link functions but no regularization.
+-  [scikit-learn] provides elastic net regularization but only limited noise distribution options.
+-  [lightning] provides elastic net and group lasso regularization, but only for
+   linear and logistic regression.
+
+[Pyglmnet] is a response to this fragmentation. The table below compares Pyglmnet with existing toolboxes.
+
+|                    | [pyglmnet] | [scikit-learn] | [statsmodels] |   [lightning]   |   [py-glm]    | [Matlab]|   [glmnet] in R |
+|--------------------|:----------:|:--------------:|:-------------:|:---------------:|:-------------:|:-------:|:---------------:|
+| **Distributions**  |            |                |               |                 |               |         |                 |
+| Gaussian           |    x       |      x         |      x        |       x         |      x        |    x    |  x              |
+| Binomial           |    x       |      x         |      x        |       x         |      x        |    x    |  x              |
+| Poisson            |    x       |                |      x        |                 |      x        |    x    |  x              |
+| Poisson (softplus)           |    x       |                |               |                 |               |         |                 |
+| Probit             |    x       |                |               |                 |               |         |                 |
+| Gamma              |    x       |                |               |                 |               |    x    |                 |
+| **Regularization** |            |                |               |                 |               |         |                 |
+| L2                 |    x       |      x         |               |       x         |               |         |                 |
+| Lasso              |    x       |      x         |               |       x         |               |         |  x              |
+| Group lasso        |    x       |                |               |       x         |               |         |  x              |
+| Tikhonov           |    x       |                |               |                 |               |         |                 |
+
+Pyglmnet implements the same algorithm described in [Friedman, J., Hastie, T., & Tibshirani, R. (2010)](https://core.ac.uk/download/files/153/6287975.pdf>) and the accompanying widely popular R package [glmnet].
```

Comment:
"...implements the algorithm described in..." or "...implements the same algorithm as described in..."